components/intl.rst

```diff
@@ -137,34 +161,57 @@ Country Names
 ~~~~~~~~~~~~~
 
 The ``Countries`` class provides access to the name of all countries according
-to the `ISO 3166-1 alpha-2`_ list of officially recognized countries and
-territories::
+to the `ISO 3166-1 alpha-2`_ list and the `ISO 3166-1 alpha-3`_ list
+of officially recognized countries and territories::
 
     use Symfony\Component\Intl\Countries;
 
     \Locale::setDefault('en');
 
+    // Indexed with alpha-2
     $countries = Countries::getNames();
     // ('countryCode' => 'countryName')
```

Comment:
perhaps mention "alpha2" here to reduce comments, e.g. `// ('alpha2Code' => 'countryName')`